app/src/main/java/org/mozilla/fenix/tabstray/viewholders/NormalBrowserPageViewHolder.kt

```diff
@@ -84,6 +84,7 @@ class NormalBrowserPageViewHolder(
         val searchTermTabGroupsAreEnabled = containerView.context.settings().searchTermTabGroupsAreEnabled
 
         val selectedTab = browserStore.state.selectedNormalTab ?: return
+        val selectedSearchGroupTabId = tabsTrayStore.state.selectedSearchGroupTabId
```

Comment:
We should also leave a comment above this line stating that it's safe to read the state directly (i.e. won't cause bugs because of the store actions processed on a separate thread) instead of observing it because this value is only set during the `initialState` of the `TabsTrayStore` being created.

Reply:
👍🏽